File: twiml/voice/dial/dial-5/dial-5.5.x.cs
using Twilio.TwiML;
using Twilio.TwiML.Voice;
using System;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var dial = new Dial(record: "record-from-ringing-dual",
            recordingStatusCallback: new Uri("www.myexample.com"));
        dial.Number("+15558675310");
        response.Append(dial);

        Console.WriteLine(response.ToString());;
    }
}

File: twiml/voice/dial/dial-5/dial-5.5.x.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;


const response = new VoiceResponse();
const dial = response.dial({
    record: 'record-from-ringing-dual',
    recordingStatusCallback: 'www.myexample.com'
});
dial.number('+15558675310');

console.log(response.toString());
